collections/_release_4_5/entry-scripting-gdextension-allow-calling-object-set-script-instance-from-gdextension.md

```diff
@@ -0,0 +1,17 @@
+---
+type: entry
+section: scripting
+subsection: gdextension
+rank: 1
+importance: 4
+anchor: allow-calling-object-set-script-instance-from-gdextension
+title: Allow calling `Object::set_script_instance()` from GDExtension
```

Comment:
This is such a small feature, I'm not totally sure we need to call it out in the release notes?

Reply:
I can remove it!